catboost/R-package/configure.ac

```diff
@@ -0,0 +1,91 @@
+# autoconf/configure script for catboost
+
+##### INIT ####################################################################
+
+AC_INIT(catboost, 0.12, nikitxskv@yandex-team.ru)
```

Comment:
Can we specify https://github.com/catboost/catboost/issues as a bug report page instead of nikitxskv@yandex-team.ru?